bin/travis/script.sh

```diff
@@ -31,7 +31,7 @@ case "$ORCA_JOB" in
   "INTEGRATED_DEV") eval "orca qa:automated-tests --sut=$ORCA_SUT_NAME $SUT_ONLY" ;;
   "CORE_NEXT") eval "orca qa:automated-tests --sut=$ORCA_SUT_NAME $SUT_ONLY" ;;
   "D9_READINESS") eval "orca qa:automated-tests --sut=$ORCA_SUT_NAME --sut-only" ;;
-  "CUSTOM") eval "orca qa:automated-tests --sut=$ORCA_SUT_NAME ${ORCA_CUSTOM_TESTS_RUN_ARGS:=}" ;;
```

Comment:
Removing the `--sut` parameter here and in install.sh is necessary to support custom jobs when no SUT is present (i.e. for acquia/drupal-recommended-project). Otherwise ORCA tries to require a SUT, and that fails when I'm trying to test a project template.

If you don't want to straight-up remove it, maybe you could make `--sut=$ORCA_SUT_NAME` a default value for `ORCA_CUSTOM_TESTS_RUN_ARGS` instead? It just needs to be something that a downstream project can override and remove completely if necessary. Or else maybe there could be a CUSTOM_NO_SUT job.